src/http/get-home-000view/index.js

```diff
@@ -87,18 +74,13 @@ async function authenticated(req) {
       let bad_connects = 0
       ticket = await data.set({ ...ticket, auth_hash, conn_hash, email_share, bad_connects })
     }
-    else {
-      // load connections
-      connections = (await fetchConnections()).filter((c) => c.from === ticket.key)
-      // inflate connections with ticket data
-      for (let i in connections) {
-        let conn = connections[i]
-        let to_data = await fetchToData(conn.to)
-        connections[i] = { ...conn, to_data}
-      }
-      //console.log(connections)
+    const connectedTicket = new ConnectedTicket(ticket)
+    const connections = await connectedTicket.connections()
+    const { csv, add_connection } = req.queryStringParameters
+    if (add_connection) {
+      const connectToTicket = Ticket.fromConnectionHash(add_connection)
+      connectedTicket.addConnection(connectToTicket)
```

Comment:
This should likely happen before collecting the connections so that it contains any new connections as well during loading.